webknossos/webknossos/dataset/view.py

```diff
@@ -571,6 +612,8 @@ def get_view(
         The default is to return the same view as the current bounding box,
         in case of a `MagView` that's the layer bounding box.
         One can supply one of the following keyword argument combinations:
+        * `relative_bbox`in Mag(1)
+        * `absolute_bbox` in Mag(1)
```

Comment:
These are called `relative_bounding_box` and `absolute_bounding_box` elsewhere.

Reply:
I changed the naming accordingly.